MetaMask/metamask-mobile
# refactor(ramp): use isBuy, isSell instead of rampType compare

## **Description**

This PR changes the comparisons `rampType === RampType.BUY` to `isBuy` and  `rampType === RampType.SELL` to `isSell` 

## **Manual testing steps**

Navigate through both Buy and Sell flows and confirm there is not changes in the UI/logic

## **Screenshots/Recordings**
n/a

## **Related issues**

n/a

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
